src/action.ts

```diff
@@ -94,6 +112,36 @@ async function handleIssueEdited(options: IssueEditedOptions) {
     page_id: pageId,
     properties: parsePropertiesFromPayload(payload),
   });
+
+  const bodyBlocks = getBodyChildrenBlocks(payload);
+
+  const existingList = (
+    await notion.client.blocks.children.list({
+      block_id: pageId,
+    })
+  ).results;
+
+  const overlap = Math.min(bodyBlocks.length, existingList.length);
+
+  await Promise.all(
+    bodyBlocks.slice(0, overlap).map((block, index) =>
+      notion.client.blocks.update({
```

Comment:
Is there anything in the api
that lets us batch these updates? And can we save calls by not updating blocks that haven't changed?

Reply:
Unfortunately, it seems like the [Notion API](https://developers.notion.com/reference/patch-page) doesn't have a way to bulk send these updates: Ones the children are added there's no way to overwrite them in one go, you have to use the delete and update endpoints for each of them. 
An alternative could be deleting the page and creating a new one, which would avoid the calls to every single block, but I guess it would break links and the order on the table (?)